apps/docs/pages/guides/auth/auth-magic-link.mdx

```diff
@@ -22,6 +22,11 @@ Setting up Magic Link logins for your Supabase application.
 
 ## Add login code to your client app
 
+<Admonition type="caution">
+When making use of Magic Links with [the PKCE Flow](https://supabase.com/blog/supabase-auth-sso-pkce#introducing-pkce) do bear in mind that links can only be used
+in the same browser that they are sent from. Consequently, a magic link sent from Chrome on Desktop will be invalid if used on a Mobile Device.
+</Admonition>
```

Comment:
Just a heads up MDX formatting + prettier can be a bit funny - currently we need to keep all text in a single line, and pad with newlines:
```suggestion
<Admonition type="caution">

When making use of Magic Links with [the PKCE Flow](https://supabase.com/blog/supabase-auth-sso-pkce#introducing-pkce) do bear in mind that links can only be used in the same browser that they are sent from. Consequently, a magic link sent from Chrome on Desktop will be invalid if used on a Mobile Device.

</Admonition>
```

otherwise we get some funky line breaks:
![image](https://github.com/supabase/supabase/assets/4133076/8b5c9c99-1d89-4d1b-9073-c922196cec59)

Reply:
👀 thanks for the advance waning!